agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/SetSubscriptionService.java

```diff
@@ -2,7 +2,5 @@
 
 import com.dke.data.agrirouter.api.service.parameters.SetSubscriptionParameters;
 
-/**
- * Service to set the subscriptions for an endpoint.
- */
+/** Service to set the subscriptions for an endpoint. */
```

Comment:
Why? Is this your formatter?

Reply:
No, it's the mvn formatter and this is the correct formatting.
It used to be wrong and was fixed in #86 .
As this branch origins from an older version of develop, it's shown as a change.
Actually, it's the same as in the current develop branch.